IBPSA/Examples/Tutorial/SimpleHouse/SimpleHouse6.mo

```diff
@@ -0,0 +1,144 @@
+within IBPSA.Examples.Tutorial.SimpleHouse;
+model SimpleHouse6 "Free cooling model"
+  extends SimpleHouse5(zone(nPorts=2),
+    mAir_flow_nominal=0.1,
+    A_win=6);
+
+  parameter Modelica.Units.SI.PressureDifference dpAir_nominal=200
+    "Pressure drop at nominal mass flow rate for air loop";
+
+  IBPSA.Fluid.Actuators.Dampers.Exponential
+                                vavDam(
```

Comment:
Could be in one line, same for fan and hexRec.

Reply:
Done